Lut arif (#802)
NEUBIAS/training-resources: _includes/display_settings/display_settings_act1.md

<h4 id="explore"><a href="#explore">Learn about contrast settings and colormaps</a></h4>

> To visualize an image there are several lookup tables to choose from.

Reviewer: ```suggestion
To visualize an image there are several colormaps to choose from.
```

Depending upon the use-case, one can be more appropriate than another.
In this activity, we will look at commonly used colormaps and discuss how and when to use them.

- Open the image [xy_8bit__nuclei_high_dynamic_range.tif](https://github.com/NEUBIAS/training-resources/raw/master/image_data/xy_8bit__nuclei_high_dynamic_range.tif)
- Adjust contrast settings to highlight certain regions

>   1. See only bright regions & observe pixel values

Reviewer: ```suggestion
  1. Highlight bright regions
```

    - Keep `max` value fixed at highest number

>     - Set `min` value to a higher number to see high pixel value regions

Reviewer: ```suggestion
    - Increase `min` value until ???
```

In fact, here I'm not sure what the aim is. Might depend on the image. Does it help here to increase the minimum value to better see the bright objects?

Author: Not really but one does see the visible objects are now smaller (see image). Maybe its a way to see which nuclei has higher intensity signal but one has to emphasize this. Alternatively, we can use another image that shows bright spots within object boundaries 
<img width="611" height="344" alt="image" src="https://github.com/user-attachments/assets/67722bf9-fcee-4f59-9894-7ee493d8537d" />

  1. Look at dim objects
    - Keep `min` value fixed at lowest number
    - Set `max` value to a lower number until you see dimmest objects
- Select appropriate colormap

>   1. Single color

Reviewer: I think this needs more work. What did you think about the idea to add discussion points for things that one should not do?
